packages/documentation/copy/en/handbook-v2/Type Manipulation/Conditional Types.md

```diff
@@ -205,15 +205,20 @@ type Bools = GetReturnType<(a: boolean, b: boolean) => boolean[]>;
 //   ^?
 ```
 
-When inferring from a type with multiple call signatures (such as the type of an overloaded function), inferences are made from the _last_ signature (which, presumably, is the most permissive catch-all case). It is not possible to perform overload resolution based on a list of argument types.
+When inferring the return type from a type with multiple call signatures, such as an overloaded function, inferences are made from the _last overload signature_, not from the [implementation signature](https://www.typescriptlang.org/docs/handbook/2/functions.html#overload-signatures-and-the-implementation-signature).
 
 ```ts twoslash
-declare function stringOrNum(x: string): number;
-declare function stringOrNum(x: number): string;
-declare function stringOrNum(x: string | number): string | number;
```

Comment:
Reading this again, I'm not sure that I like this change. This last one in the example is an overload, not an implementation signature; it's not uncommon for people to define these catch-all overloads at the bottom.

Perhaps it'd be better to leave the existing text as-is, and then add a new comment at the bottom that says that implementation signatures are not part of a function's type, and therefore are not considered?